### PR TITLE
feat: adding actions API

### PR DESCRIPTION
**Pull request by erickzanardo**
Adds a new API called `actions` to enable methods to called from Dashbooks UI.

### CODE REVIEW

**Review by tyemy — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -73,6 +73,29 @@ void main() {
 }
 ```
 
+## Actions
+
+Dashbook also provides a way for easily calling methods from its UI, these callbacks can be used for more complex examples which demands user interaction.
+
+For example, a `Dialog` is something that isn't direclty rendered on a page, but rather show upon an action, an example for a `CustomDialog` widget could be achieved on Dashbook by using the following code:
```
Comment (spydon) [inline README.md:80]: Maybe split up this line so that it isn't so long?

Comment (spydon) [reply README.md:80]: Directly* shows*

File: lib/src/widgets/keys.dart [hunk 2]
```diff
@@ -1,6 +1,8 @@
 import 'package:flutter/material.dart';
 
 const kPropertiesIcon = Key('properties_icon');
+const kActionsIcon = Key('actions_icon');
+const kActionsCloseIcon = Key('actions_close_icon');
```
Comment (spydon) [inline lib/src/widgets/keys.dart:5]: Not related to this PR, but something to think about 
https://dart.dev/guides/language/effective-dart/style#dont-use-prefix-letters

**Review by spydon — APPROVED** [review]

**Review by renancaraujo — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (erickzanardo) [reply README.md:80]: I need to split this whole file :P

Fixed the typos, thanks!

**Pull request merged**